bin/sdl_make_tracklooper

```diff
@@ -63,17 +67,22 @@ if [ -z ${MAKECUTVALUES} ]; then MAKECUTVALUES=false; fi
 if [ -z ${PRIMITIVE} ]; then PRIMITIVE=false; fi
 if [ -z ${TCPLSTRIPLETS} ]; then TCPLSTRIPLETS=false; fi
 if [ -z ${DONTUSENN} ]; then DONTUSENN=false; fi
-if [ -z ${ONLYCUDABACKEND} ]; then ONLYCUDABACKEND=false; fi
-if [ -z ${ONLYCPUBACKEND} ]; then ONLYCPUBACKEND=false; fi
+if [ -z ${CPUBACKEND} ]; then CPUBACKEND=false; fi
+if [ -z ${CUDABACKEND} ]; then CUDABACKEND=false; fi
+if [ -z ${ROCMBACKEND} ]; then ROCMBACKEND=false; fi
+if [ -z ${ALLBACKENDS} ]; then ALLBACKENDS=false; fi
 if [ -z ${NOPLSDUPCLEAN} ]; then NOPLSDUPCLEAN=false; fi
 if [ -z ${PRINTWARNINGS} ]; then PRINTWARNINGS=false; fi
 if [ -z ${PTCUTVALUE} ]; then PTCUTVALUE=0.8; fi
 
-# If using both -G and -C, -G takes priority
-if [ "${ONLYCUDABACKEND}" == true ] && [ "${ONLYCPUBACKEND}" == true ]; then
-  echo "WARNING: -C and -G flags should not be used together."
-  echo "         Only the CUDA backend will be compiled"
-  ONLYCPUBACKEND=false
+# Default to only CPU and CUDA backends
+if [ "${CPUBACKEND}" == false ] && [ "${CUDABACKEND}" == false ] && [ "${ROCMBACKEND}" == false ]; then
+  CPUBACKEND=true
+  CUDABACKEND=true
+elif [ "${ALLBACKENDS}" == true ]; then
```

Comment:
isn't `ALLBACKENDS = true` set by `A` flag (in absence of `GCR`)  going to select the first `if` and will not make it to the `elif`?